src/model/questionGenerator.test.js

```diff
@@ -0,0 +1,54 @@
+import generateQuestion from './questionGenerator';
+import { charactersTestData, quotesTestData, deathsTestData } from './questionGeneratorTestData';
+
+describe("generateQuestion", () => {
+  test("if we ask for characters we should get object with answers, correctAnswer and questionObject", async () => {
+    global.fetch = jest.fn(()=> Promise.resolve({json: () => charactersTestData,
+      charactersTestData}));
+    const {answers, correctAnswer, questionObject} = await generateQuestion('characters',[]);
+    expect(answers).not.toBeUndefined();
+    expect(correctAnswer).not.toBeUndefined();
+    expect(questionObject).not.toBeUndefined();
+  });
+
+  test("if we ask for quotes we should get object with answers, correctAnswer and questionObject", async () => {
+    global.fetch = jest.fn(()=> Promise.resolve({json: () => quotesTestData,
+      quotesTestData}));
+    const {answers, correctAnswer, questionObject} = await generateQuestion('quotes',[]);
+    expect(answers).not.toBeUndefined();
+    expect(correctAnswer).not.toBeUndefined();
+    expect(questionObject).not.toBeUndefined();
+  });
+
+  test("if we ask for characters we should get object with answers, correctAnswer and questionObject", async () => {
+    global.fetch = jest.fn(()=> Promise.resolve({json: () => deathsTestData,
+      deathsTestData}));
+    const {answers, correctAnswer, questionObject} = await generateQuestion('deaths',[]);
+    expect(answers).not.toBeUndefined();
```

Comment:
https://jestjs.io/docs/expect#tobetruthy

Reply:
Niżej gdzieś używałem toBetTruthy, ale tutaj nie wpadłem na to, żeby użyć. Już zmienione :)